spec/features/user_changes_list_records_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+feature "user changes list records" do
+  scenario "updates successfully" do
+    user = create(:user)
+    list = create(:list, user: user)
```

Comment:
Can we specify the name here? Given that we're going to change it, we should ensure that by some RANDOM chance the factory didn't name it "New List Name"